tests/ui/app/reviews.js

```diff
@@ -132,7 +136,7 @@ casper.test.begin('Test review button hidden when logged in and cannot rate', {
             helpers.fake_login();
         });
 
-        helpers.waitForPageLoaded(function() {
+        waitForReviewsLoaded(function() {
```

Comment:
`.app-reviews` doesn't exist until the reviews have been loaded which is in a [`defer` block](https://github.com/mozilla/fireplace/blob/615642bf2c1cbc5c06d19a25ecd86d5a86b5bafe/src/templates/app/index.html#L154).